src/management/api/creation/steps/api-creation-step1.html

```diff
@@ -71,6 +71,18 @@
             </div>
           </md-input-container>
         </div>
+
+        <div layout-gt-sm="row">
```

Comment:
it could be could to disable the control or to hide this group when no group available ? 

It is a little weird to be allowed to click on the selector and open a blank list.
wdyt ?

![image](https://user-images.githubusercontent.com/47851994/106444442-0f488580-647e-11eb-97f3-d4c11d30705f.png)